website/addons/wiki/static/pagedown-ace/Markdown.Editor.js

```diff
@@ -59,7 +60,7 @@ var Range = ace.require('ace/range').Range;
         undo: "Undo -",
         redo: "Redo -",
 
-        help: "Markdown Editing Help"
+        help: "Wiki Syntax Help"

```

Comment:
The modal title is in another file. I'll change it there, but I think this is a pop-up for hover which is more in line with the code above it.